common/thorhelper/thorxmlwrite.cpp

```diff
@@ -246,7 +246,10 @@ void CommonXmlWriter::outputXmlns(const char *name, const char *uri)
 {
     StringBuffer fieldname;
     if (!streq(name, "xmlns"))
-        fieldname.append("xmlns:");
```

Comment:
Yes, I was misusing this method. My original change to this method is not necessary.